tests/build_tests/super_errors/expected/method_arity_mismatch.res.expected

```diff
@@ -8,5 +8,8 @@
   4 [2m│[0m }
   5 [2m│[0m 
 
-  This function has type (int, int) => unit
-  It is applied with [1;31m1[0m argument but it requires [1;33m2[0m.
\ No newline at end of file
+  This function call is incorrect.
+  The function has type:
+  (int, int) => unit
+  
+  It is called with [1;31m1[0m argument but requires [1;33m2[0m.
```

Comment:
For some reason the new error mechanism does not pick up this specific test. It's an object method call, looks like this:
```rescript
let f = obj => {
  obj["hi"](1, 2)
  obj["hi"](1)
}
```

Couldn't figure out why it wasn't picked up, should've been caught with the too few unlabelled arguments check. Maybe @cristianoc has ideas?

Either way, it's not worse than before, so we can definitely ship this and figure out how to improve it later.

Reply:
This was resolve right?

Reply:
No, not yet. But it's not important now, can be investigated as a follow up.